target/linux/ramips/dts/mt7620a_edimax_ew-747x.dtsi

```diff
@@ -202,7 +202,7 @@
 };
 
 &gsw {
-	mediatek,port5 = "gmac";
+	mediatek,port4-gmac;
```

Comment:
What was going on here, that it was previously setting mediatek,port*5* ?

Reply:
`mediatek,port5` never existed as a setting that the driver recognized, the author assumed that since there is a property for port4 there is also one for port5...and it was not caught during review

Reply:
But then, this should probably be dealt with in a separate commit, as people looking at it in the future might have the same question.

Reply:
alternatively, I can just delete the node here, since under the ethernet node there is only `port@5`

Reply:
I deleted the node here